packages/react-vapor/src/components/actions/TriggerAction.tsx

```diff
@@ -21,11 +21,9 @@ export const CONFIRM_LABEL: string = 'Are you sure?';
 export class TriggerAction extends React.Component<ITriggerActionProps, any> {
     private onTriggerAction() {
         const confirmData: IConfirmData = this.props.action.requiresConfirmation;
-
         if (confirmData && this.props.onTriggerConfirm) {
-            const confirmLabel: string = this.props.confirmLabel || CONFIRM_LABEL;
+            const confirmLabel: string = this.props.action.requiresConfirmation.confirmLabel || CONFIRM_LABEL;

```

Comment:
This is a breaking change, we should do 
```suggestion
            const confirmLabel: string = this.props.confirmLabel || this.props.action.requiresConfirmation.confirmLabel || CONFIRM_LABEL;
```

Reply:
:+1: